src/documents/views.py

```diff
@@ -794,17 +796,41 @@ class StatisticsView(APIView):
 
     def get(self, request, format=None):
         documents_total = Document.objects.all().count()
-        if Tag.objects.filter(is_inbox_tag=True).exists():
-            documents_inbox = (
-                Document.objects.filter(tags__is_inbox_tag=True).distinct().count()
+
+        inbox_tag = Tag.objects.filter(is_inbox_tag=True)
+
+        documents_inbox = (
+            Document.objects.filter(tags__is_inbox_tag=True).distinct().count()
+            if inbox_tag.exists()
+            else None
+        )
+
+        document_file_type_counts = (
+            Document.objects.values("mime_type")
+            .annotate(mime_type_count=Count("mime_type"))
+            .order_by("-mime_type_count")
+            if documents_total > 0
+            else 0
+        )
+
+        character_count = (
+            sum(
```

Comment:
This sum can be done via query:
```python
>>> q = Document.objects.annotate(characters=Length("content")).aggregate(Sum("characters"))
>>> q
{'characters__sum': 1084}
```
It will be None if there's no documents though that looks handled already